lib/player.rb

```diff
@@ -0,0 +1,43 @@
+class Player
+
+	attr_reader :type, :current_score
+	
+	def initialize(type = "Human")
+		raise "Bad player type specified" unless (type == "Human" || type == "Computer")
```

Comment:
you can also create an array of valid players - `["Human", "Computer"]` - and check if `type` is valid using `includes?`